packages/docs/docs/docs/deploying.md

```diff
@@ -87,112 +91,73 @@ contract MyContract is Initializable {
-a personal blockchain for Ethereum development that you can use to develop
-your contracts. To install it run:
+You're now ready for the initial deployment of the project. To do this, however, you'll need a blockchain network on which to deploy it. For this example, you can use [Ganache](https://truffleframework.com/docs/ganache/quickstart), which is a personal blockchain for Ethereum development that you can use to deploy your contracts. To install it, run:
 
 ```console
 npm install -g ganache-cli
 ```
 
-To start working with it, open a separate terminal and run:
+Open a separate terminal and run Ganache:
 
 ```console
 ganache-cli --port 9545 --deterministic
 ```
 
-Once we have done that, let's go back to the original terminal and
-run the following command:
+Now run `zos create`:
 
 ```console
-npx zos session --network local --from 0x1df62f291b2e969fb0849d99d9ce41e2f137006e --expires 3600
+$ npx zos create
 ```
 
-The `session` command starts a session to work with a desired network.
-In this case, we are telling it to work with the `local` network with the
-`--network` option, and also setting a default sender address for the
-transactions we will run with the `--from` option. Additionally, the
-`expires` flag allows us to indicate the session expiration time in seconds.
+Most of the ZeppelinOS commands, such as `create`, are interactive as of version 2.3 and will prompt you for the needed information. In this case, the `create` command will prompt you, and you can use **Enter** to select the defaults as follows:
 
-Now that everything has been setup, we are ready to deploy the project.
-To do so simply run:
+`Choose a contract:` **MyContract**
+`Select a network:` **local**
+`Do you want to run a function after creating the instance?` **Y**
+`Select a method:` **initialize**
 
-```console
-npx zos push
-```
+When it prompts you for initial variable values, you can give it any **int** and **string** such as:
 
-This command deploys `MyContract` to the specified network and prints its
-address. If your project added other
-contracts (using the `add` command) they would be deployed as well.
+`_x`: **42**
+`_s`: **hitchhiker**
 
-The `push` command also creates a `zos.dev-<network_id>.json` file with all the
-information about your project in this specific network, including the addresses of the
-deployed contract implementations in `contracts["MyContract"].address`.
-You can read more about this file format in the
-[configuration files](configuration.md#zos-network-json) section.
+The interactive command does the following:
 
-If a previous `zos.dev-<network_id>.json` was already created, the `push`
-command will just update said file with your last changes. For example, if you
-change the source of `MyContract` and call `zos push` again,
-ZeppelinOS will deploy a new version of `MyContract`
-and replace the entry in the json file's `contracts["MyContract"].address`.
+* Invokes `create` to compile your contract.
```

Comment:
```suggestion
* Uses truffle to compile your contract.
```
Note that the `create` operation just by itself does not compile.

Reply:
Made this change.